name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/SecuritiesChart.java

```diff
@@ -1396,15 +1406,13 @@ private void addDividendMarkerLines(ChartInterval chartInterval)
             if (chartConfig.contains(ChartDetails.SHOW_DATA_LABELS))
             {
                 customPaintListeners.add(event -> {
-                    FontMetrics fontMetrics = event.gc.getFontMetrics();
+                    Color defaultForeground = Colors.theme().defaultForeground();
+                    event.gc.setForeground(defaultForeground);
 
                     // Three levels of the label
                     int[] labelExtendX = new int[3];
 
                     int symbolSize = border.getSymbolSize();
-                    int labelHeight = fontMetrics.getHeight();
-                    int halfLabelHeight = labelHeight / 2;
```

Comment:
Just out of curiosity, was there a specific reasons why the fontMetrics did not work?
I think the idea was that the label height never changes.
(But the current code is also okay because we anyway need to get the text extend)

Reply:
All labels in the Security Chart are created without fontMetrics, so it is more consistent if we keep this as long as it has no effect. 😎